### ISSUE TITLE
Does number of confirmed cases include #deaths and #recovered?

### ISSUE THREAD

**Issue by Ling-Hong-symphonyrm** [CSSEGISandData/COVID-19#1271]
Is it true that #Confirmed cases =  #infectious + #death + #recovered? 
Or they are separate numbers?

**Comment by Maanuj-Vora** [comment]
Yes, confirmed cases does include deaths and recovered. To get the active cases, do the following

active_cases = (confirmed_cases - deaths) - recovered
